examples/lora/lorawan/basic-demo/main.go

```diff
@@ -9,9 +9,11 @@ import (
 	"tinygo.org/x/drivers/examples/lora/lorawan/common"
 	"tinygo.org/x/drivers/lora"
 	"tinygo.org/x/drivers/lora/lorawan"
+	"tinygo.org/x/drivers/lora/lorawan/region"
 )
 
 const (
+	debug                       = true
```

Comment:
Make this into 

```go
var Debug string
``` 

and you can use 

```
tinygo flash -ldflags="-X main.Debug=true" -target lorae5 ./examples/lora/lorawan/basic-demo/
```

You will also need to change the condition check, see additional comment.